src/ipv6/ipv6.c

```diff
@@ -75,6 +75,22 @@ static RTE_DEFINE_PER_LCORE(struct inet_stats, ip6_stats);
         this_ip6_stats.__f__##octets += (val); \
     } while (0)
 
+#ifdef CONFIG_DPVS_MBUF_DEBUG
+static inline void ip6_show_hdr(const char *func, struct rte_mbuf *mbuf)
```

Comment:
CONFIG_DPVS_MBUF_DEBUG is not a good name, because we didn't dump mbuf but just ipv4/ipv6 header

Reply:
How about using "CONFIG_DPVS_IP_HEADER_DEBUG"?

Reply:
I think it's fine.

Reply:
Done.